srfi-tools/generate.scm

```diff
@@ -5,6 +5,169 @@
 (define (faq-anchor-template description name)
   `(a (@ (href ,(concat "/srfi-faq.html#" name))) ,description))
 
+;; Format string for srfi-NNN.html.
+;; TODO: Convert to SXML.
+(define srfi-html-template "<!DOCTYPE html>
+<html lang=\"en\">
+  <head>
+<!--
+SPDX-FileCopyrightText: ~a ~a
+SPDX-License-Identifier: MIT
+-->
+    <meta charset=\"utf-8\">
+    <title>SRFI ~a: ~a</title>
+    <link href=\"/favicon.png\" rel=\"icon\" sizes=\"192x192\" type=\"image/png\">
+    <link rel=\"stylesheet\" href=\"https://srfi.schemers.org/srfi.css\" type=\"text/css\">
+    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">
+    <style>
+     small { font-size: 14px; vertical-align: 2px; }
+     body  { line-height: 24px; }
+     pre  { font-family: inherit; line-height: 20px; }
+    </style>
+  </head>
+  <body>
+    <h1><a href=\"https://srfi.schemers.org/\"><img class=\"srfi-logo\" src=\"https://srfi.schemers.org/srfi-logo.svg\" alt=\"SRFI surfboard logo\" /></a>~a: ~a</h1>
+
+<p>by ~a</p>
+
+<h2 id=\"status\">Status</h2>
+
+    <p>
```

Comment:
Please drop the body of the Status section, or just include the example from srfi-template.html:

<p style="color: red;">For editor's use only.  Please do not edit this section.</p>

<p>??? the draft/final/withdrawn status of the SRFI, information on how
to subscribe to its mailing list, and important dates in its history.
The editor will add this section.</p>